lib/elixir_sense/core/metadata.ex

```diff
@@ -58,6 +60,31 @@ defmodule ElixirSense.Core.Metadata do
     end
   end
 
+  def get_position_to_insert_alias(%__MODULE__{} = metadata, line) do
+    env = get_env(metadata, line)
+    module = env.module
+
+    cond do
+      Map.has_key?(metadata.first_alias_positions, module) ->
+        Map.get(metadata.first_alias_positions, module)
+
+      Map.has_key?(metadata.moduledoc_positions, module) ->
+        Map.get(metadata.moduledoc_positions, module)
+
+      true ->
+        mod_info = Map.get(metadata.mods_funs_to_positions, {env.module, nil, nil})
+
+        case mod_info do
+          %State.ModFunInfo{positions: [{line, column}]} ->
+            # Hacky :shrug
+            {line + 1, column - 10 + 2}
```

Comment:
What do these numbers represent?

I think you could extract them to variables, just to get the [magic-number](https://wiki.c2.com/?MagicNumber) look out of it:

```suggestion
            line_offset = 1
            column_offset = -8
            
            {line + line_offset, column + column_offset}
```